nativeauthenticator/nativeauthenticator.py

```diff
@@ -166,13 +166,22 @@ def is_password_strong(self, password):
             checks.append(not self.is_password_common(password))
 
         return all(checks)
-
-    def get_or_create_user(self, username, pw, **kwargs):
-        username = self.normalize_username(username)
-        user = UserInfo.find(self.db, username)
+        
+    def get_user(self, username):
+        user = UserInfo.find(self.db, self.normalize_username(username))
```

Comment:
If I am not mistaken, if the user doesn't exist it already returns None, so we can remove the following if and just return the result of this search